pkg/api/datacenter_cache.go

```diff
@@ -6,16 +6,24 @@ import (
 	"fmt"
 	"net/http"
 	"sort"
+	"strconv"
 	"sync"
 
 	"github.com/prometheus/client_golang/prometheus"
 )
 
 // Datacenter models a single datacenter as returned by the Fastly API.
 type Datacenter struct {
-	Code  string `json:"code"` // Prometheus label is "datacenter" to make grouping at query time less tedious
-	Name  string `json:"name"`
-	Group string `json:"group"`
+	Code        string      `json:"code"` // Prometheus label is "datacenter" to make grouping at query time less tedious
+	Name        string      `json:"name"`
+	Group       string      `json:"group"`
+	Coördinates Coördinates `json:"coordinates"`
```

Comment:
Minor question (ok, fine, pedantic), should we be using umlats/diaeresis in predominantly english based code and naming?

Coördinates vs Coordinates

Other then the new yorker (https://www.newyorker.com/culture/culture-desk/the-curse-of-the-diaeresis) it doesn;t seem to be common parlance ;)

Reply:
It's just my thing :)